ch_pipeline/processing/daily.py

```diff
@@ -141,10 +141,23 @@
       params:
         exclude_intracyl: true
 
-    # Identify individual baselines with much lower weights than expected and
-    # mask out the entire time-freq sample
-    - type: draco.analysis.flagging.ThresholdVisWeightTime
+    # Identify individual baselines with much lower weights than expected
+    - type: draco.analysis.flagging.ThresholdVisWeightBaseline
+      requires: manager
       in: tstream
+      out: full_bad_baseline_mask
+      params:
+        average_type: "median"
+        absolute_threshold: 1e-7
+        relative_threshold: 1e-5
+        pols_to_flag: "copol"
+        save: true
+        output_name: "bad_baseline_mask_{{tag}}.h5"
```

Comment:
I think we elected not to save these out, right?

Reply:
Oops sorry, I completely misread your Slack message! force-push incoming...

Reply:
🚅